paper/paper.md

```diff
@@ -0,0 +1,333 @@
+We had thirty-nine learners of various skill levels from several different
+countries, all of whom provided very positive reviews of the workshop.
+By the end of the workshop learners on average felt more confident writing programs, using programming to work with data, overcoming problems while programming, and searching for answers to technical questions online (n = 14 survey respondents; pre- and post-workshop surveys).
+All attendees who filled out the post-workshop survey (n = 19) would also recommend the workshop to others.
+
+### Virtual Workshop Reflection
+
+We credit the success of our first virtual workshop in large part due to the
+curriculum structure and content, as well as the instructors and helpers
+involved.
+However, we also believe that the following helped make the workshop as
+smooth as possible:
+
+1. We suggested that learners have Zoom and RStudio (or the Unix shell) open side-by-side on their computer to minimize toggling between different panels.
+1. We used Slack for communication among instructors and helpers, as well as between helpers and learners.
+Learners asked questions in a group Slack channel where helpers could respond.
+This allowed us to answer the vast majority of learner questions and bugs quickly, clearly, and efficiently without disrupting the lesson or moving the learner to a breakout room.
+Furthermore, Slack worked much better than the Zoom chat as questions can be answered in threads, are visible to all learners regardless of whether they are connected to Zoom at the time, and  don't get lost as easily.  
+1. When a learner needed more help than was possible on Slack, a helper and the
+learner entered a breakout room together to troubleshoot.
+However, we tried to minimize this option as then the learner misses content
+covered in the main room.
+
+# Acknowledgements
+
+We thank The Carpentries organization for providing instructor training,
+workshop protocols, and the open-source Software Carpentry curriculum, upon
+which this curriculum is based. We also thank them for allowing us to use the
+pre- and post-workshop survey results in this manuscript.
+The Carpentries is a fiscally sponsored project of Community Initiatives, a registered 501(c)3 non-profit organisation based in California, USA.
+
+We thank Victoria Alden and Scott Martin for assisting us in organizing and
+advertising our pilot workshop.
+We also thank Shelly Johnson for volunteering as a helper at the workshop.
+
+We thank the learners who participated in the workshop, provided feedback, and completed the surveys.
+
+# Funding
+
+Salary support for PDS came from NIH grant R01CA215574.
+KLS received support from the NIH Training Program in Bioinformatics (T32
+GM070449).
+ZL received support from the National Science Foundation Graduate Research
+Fellowship Program under Grant No. DGE 1256260.
+Any opinions, findings, and conclusions or recommendations expressed in this
```

Comment:
```suggestion
Any opinions, findings, conclusions, or recommendations expressed in this
```

Reply:
this might be official language that @zenalapp has to use

Reply:
Yeah I'm just going to leave it because that's what they tell us to say.